README.md

```diff
@@ -1,3 +1,25 @@
+Task 3 what was done:
+1. Task 3.1  all criteria are followed. The [link](https://d359ghg4pp7ul2.cloudfront.net/) to the CloudFront to check if the api integrated. 
+2. Task 3.2  all criteria are followed.
+3. Additional (optional) tasks:
```

Comment:
Hey, to get 100 you just need to add Swagger docs. The simple solution is to use a plugin called `serverless-auto-swagger`. Check out that thread from our course channel https://teams.microsoft.com/l/message/19:73dafd02206d421896229364386d7bd2@thread.skype/1677417048461?tenantId=b41b72d0-4e9f-4c26-8a69-f949f367c91d&groupId=a91a4b92-98b1-4d0d-9e21-bd706bd68ed5&parentMessageId=1677417048461&teamName=EPAM%20JS%20EDU%20L%26D&channelName=CloudX%20AWS%20Practitioner%20for%20JS&createdTime=1677417048461&allowXTenantAccess=false.

Otherwise please let me know if you are OK with 96. I'll mark as passed and complete the review as it is.